content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>I wouldn't keep the code comments, but rather create another cell with the unused options or explain them in markdown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Sorry, that was a typo that I missed in the cleaned file. </p>